<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta charset="utf-8" />
		<title>2660</title>
	</head>
	<body id="x2660" lang="sl-SI">
		<sidebar id="_idContainer000" class="Basic-Text-Frame">
			<p class="slug"><bold class="strong">@publisher.id:</bold> 2660</p>
			<p class="slug"><bold class="strong">@translation</bold><bold class="strong">:</bold> sl, <span class="primary-language">en</span></p>
			<p class="slug"><bold class="strong">@discipline-en:</bold> Microbiology and immunology, Stomatology, Neurobiology, Oncology, Human reproduction, Cardiovascular system, Metabolic&#160;and&#160;hormonal&#160;disorders, Public&#160;health (occupational medicine), <span class="discipline-en">Psychiatry</span></p>
			<p class="slug"><bold class="strong">@article-type-en:</bold> Editorial, Original scientific article, Review article, Short scientific article, <span class="article-type-en">Professional article</span></p>
			<p class="slug"><bold class="strong">@running-header:</bold> <span class="running-header">Epigenetics and suicidal behaviour</span></p>
			<p class="slug"><bold class="strong">@reference-en:</bold> <span class="reference-en">Zdrav Vestn | September – October 2018 | Volume 87</span></p>
		</sidebar>
		<sidebar id="_idContainer001" class="meta-wrapper">
			<p class="aff lang-en">Institute of Biochemistry, Faculty of Medicine, University of Ljubljana, Ljubljana, Slovenia</p>
			<p class="label">Correspondence:</p>
			<p class="correspondence">Katarina Uršič, e:&#160;katarina.ursic@mf.uni-lj.si</p>
			<p class="label">Key words:</p>
			<p class="keywords lang-en">epigenetic modifications; suicide; DNA methylation; miRNA;histone modification</p>
			<p class="history">Received: 13. 10. 2017 Accepted: 26. 4. 2018</p>
		</sidebar>
		<article id="_idContainer002" class="article">
			<h1 class="title lang-en" lang="en-US">Epigenetics and suicidal behaviour</h1>
			<p class="author" lang="en-US"><span class="given-name">Katarina</span> <span class="surname">Kouter</span>, <span class="given-name">Alja</span> <span class="surname">Videtič Paska</span></p>
			<h1 class="abs">Abstract</h1>
			<p class="abs">Suicide is a well-defined public health problem. Slovenia is ranked as one of the European leading countries regarding suicide rate. So far, many neurotransmitter systems have been studied on genetic and biochemical levels. In recent years, focus shifted towards research in the field of epigenetics, which includes noncoding RNA (miRNA), DNA methylation, and post-translational histone modification. miRNAs are short, 19 to 25 nucleotides long single-stranded RNA molecules that bind specifically to mRNA and decrease the expression of targeted genes. Suicide victims showed changes in the expression of miRNA associated with brain neurotrophic factor (BDNF) and its receptor (TrkB), transcription and growth factors, and the polyamine system. DNA methylation is the process of adding methyl group to the 5C site of the cytosine and it results in a decrease of gene transcription. Studies of global methylation identified changes in the patterns of promoter methylation in suicide victims. Many promoters belonged to genes involved in cognitive functions. Looking at candidate genes, changes in methylation pattern were observed at genes for BDNF, TrkB, ribosomal RNA, and the type 2A serotonin receptor promoter region. Histones are the core building blocks of chromatin, representing the protein base around which the DNA is coiled and forming nucleosomes. Histones consist of a globular core part and an N-terminal tail of different lengths, which can be post-translationally modified. Most common modifications are lysine and arginine acetylation and methylation, serine and threonine phosphorylation, lysine ubiquitination and sumoylation, and ADP-ribosylation. Histone modifications of TrkB and polyamine system were observed in suicide victims. Numerous studies thus link epigenetic mechanisms to suicidal behaviour.</p>
			<p class="cite-as"><bold class="strong">Cite as:</bold> Kouter K, Videtič Paska A. [Epigenetics and suicidal behaviour]. Zdrav Vestn. 2018;87(9–10):417–28.</p>
			<p class="doi"><bold class="strong">DOI:</bold> 10.6016/ZdravVestn.2660</p>
			<h1><span>1 </span>Introduction</h1>
			<p>Mental health is defined as a state of well-being in which every individual realizes his/her own potential, can cope with the normal stresses of life, can work productively, and is able to make a contribution to the community. Mental disorders represent a major world public health challenge. It is estimated that at least one-tenth of people suffer from one of mental health problems, therefore these disorders are the second leading cause of morbidity after ischaemic heart diseases. Individuals with mental disorders have higher mortality rates, which are mostly due to chronic diseases, such as cardiovascular and respiratory tract diseases, diabetes and suicide, which develop because of changed lifestyles and poor health care. Persons with depression and shizophrenia show a 40 to 60 higher risk of premature death&#160;<span class="xref">(1)</span>. Borderline personality disorder, depression, shizophrenia and bipolar disorder are associated with the highest risk for suicidal behaviour. Up to 15 % of persons diagnosed with at least one mental disorder die by suicide&#160;<span class="xref">(2</span><span class="xref">)</span>. Suicide is a complex phenomenon. One person in the world dies due to suicide every 40 minutes, and the World Health Organisation predicts further increases in suicide rates in the future. The suicide index (no.of suicide victims per 100,000 inhabitants) places Slovenia among the countries with the world‘s highest suicide rate. In 2016, 317 persons in Slovenia died due to suicide. Extensive research of neurotransmitter systems has been conducted at the biochemical and genetic levels. Most genetic studies have investigated serotonin, dopamine and noradrenergic systems, but in recent years, increasing attention has been given to the field of epigenetics&#160;<span class="xref">(3-7)</span>. Studies of Slovene cohorts confirmed the association between the genetic background and neurotransmitter systems and suicide&#160;<span class="xref">(8-14)</span>.</p>
			<h1><span>2 </span>Epigenetics</h1>
			<p>Mental disorders are very complex conditions, influenced by a combination of inherited and environmental factors. Genetics can only partly explain the risks for their development. Epigenetics is a link that bridges environmental and genetic factors. It is the study of changes in gene expression that occur without a change in the DNA sequence and are meiotically or mitotically heritable (intergenerational epigenetic inheritance). Epigenetic mechanisms which regulate gene expression at different levels involve noncoding DNA, DNA methylation and posttranslational modification of histones&#160;<span class="xref">(15)</span>. Epigenetic processes play an important role in the development and differentiation of cells during embryogenesis, but also during adulthood. Epigenetic mechanisms allow cells to retain their identity and adapt to the environment, and show a high degree of tissue specificity. The epigenetic state of cells is thus partly reversible and may change under the influence of environmental factors&#160;<span class="xref">(16)</span>.</p>
			<h2><span>2.1 </span>MicroRNA</h2>
			<p>Non-coding RNAs can be divided into two groups: long non-coding and short non-coding RNAs, and microRNAs (miRNAs) belonging to the latter group. miRNAs are short, 19 to 25 nucleotides long single-stranded RNA molecules that bind specifically to miRNA and decrease the expression of targeted genes&#160;<span class="xref">(17)</span>. The first miRNA was discovered more than 20 years ago. Ambros et al (1993), who investigated developmental phases of <italic class="em">C.elegans </italic>nematode, noted that lin-4 through inappropriate repression of LIN-14 protein disturbs temporal timing of the nematode development. Further studies showed that the product of lin-4 gene was not a protein miRNA but two short RNA sequences, 22 to 61 nucleotides in length. The shorter one is known as the first miRNA lin-4&#160;<span class="xref">(18)</span>. Several years elapsed to the discovery of the second miRNA let-7, which is also involved in the timing of nematode developmental events&#160;<span class="xref">(19)</span>. Thanks to the evolutionary conservation of protein region and homology, the identification of let-7 allowed for discovery of miRNAs in other organisms. A large number of miRNAs identified in animals and plants are stored in the miRBase database available for online searching (<a href="http://www.mirbase.org/">http://www.mirbase.org</a>). The database comprises 2,588 human mature miRNA sequences&#160;<span class="xref">(20)</span>. Though they play the same role in post-translational gene regulation, miRNAs present in plants and animals probably follow two separate mechanisms that must have evolved independently during evolution. So, they differ in the enzymes involved, miRNA binding site and mechanism of action. The description of miRNAs therefore refers to animal miRNA&#160;<span class="xref">(21)</span>.</p>
			<p>Mature miRNAs are of endogenous origin. They may be encoded in intrones or exones of other genes and share the same promoter. miRNAs may be encoded in sets in a so-called polycistrionic transcript, which provides information on the origin of several separate miRNAs&#160;<span class="xref">(</span><span class="xref">22)</span>. Over one-half of human protein-coding genes have at least one potential binding site for miRNA&#160;<span class="xref">(23)</span>.</p>
			<p>The miRNA gene is initially transcribed by RNA-polymerase II, which binds to DNA, encoding miRNA transcript, into a primary miRNA (pri-miRNA), which is more than 1,000 bases long (in this secondary structure the strand coils,which leads to pairing of adjacent bases in the same strand). In the nucleus, pri-miRNA is cleaved by the enzyme Drosha into precursor RNAs (pre-miRNAs) 70 bases in length. Pre-miRNAs are transported from the nucleus into the cytoplasm via Exportin 5. There, a miRNA duplex, 45 bases in length, made up of miRNA and its complementary strand miRNA*, is created by the enzyme Dicer&#160;<span class="xref">(22)</span>. A RNA-induced silencing complex (RISC) is formed by the Argonaute family proteins and the miRNA duplex.There follows the unwiding of the duplex and production of mature miRNA, usually with decomposition of its complementary sequence&#160;<span class="xref">(</span><span class="xref">24)</span>. Yet, the strand complementary to miRNA may be conserved, and may assume a regulatory function&#160;<span class="xref">(25)</span>. Differences between miRNA and its complementary strand miRNA* are determined by their chemical and physical characteristics: miRNA contains a thermodynamically less stable 5‘ end and begins with the preferred nucleotide uracil&#160;<span class="xref">(26)</span>. A mature miRNA- RISC complex binds to the 3‘- untranslated region of target mRNA&#160;<span class="xref">(22)</span>.</p>
			<p>The Argonaute family proteins which constitute the active part of RISC can cleave mRNA, promote binding of translational repressors or inhibit translation. Binding of RISC-miRNA complex to miRNA is not necessarily complementary. A single miRNA can thus regulate up to 100 different mRNAs. The so-called miRNA »seed region«,i.e. nucleotides 2–7, is involved in target recognition and has a key role in miRNAs binding to their target mRNAs&#160;<span class="xref">(23)</span>.</p>
			<p>miRNAs themselves are subject to regulation. Single nucleotide polymorphisms (SNP) in the encoding miRNA region can change the binding specificity of miRNA or affect the process of miRNA maturation&#160;<span class="xref">(27)</span>. Nucleoside modifications at the 5‘-end of pre-miRNA may lead to impaired RNA processing by the Dicer enzyme (e.g. 2‘- 0-methylation, which occurs when a methyl group is added to the 2‘ hydroxyl of the ribose moeity of a nucleoside)&#160;<span class="xref">(28)</span>.</p>
			<p>miRNA expression shows a high degree of temporal and spatial dynamics. Alterations of miRNA expression pattern can therefore lead to various pathological conditions, including mental disorders&#160;<span class="xref">(29)</span>.</p>
			<h2><span>2.2 </span>DNA methylation</h2>
			<p>DNA methylation is a process by which methyl groups are added to the 5C site of cytosine. It is catalysed by DNA-methyltransferases and represents the most frequent epigenetic change in mammals. The DNA-methyltransferases group comprises DNA-methyltransferases 1, 3A and 3B. DNA-methyltransferase 1 is responsible for maintaining the existing methylation pattern, while DNA-methyltransferases 3A and 3B are involved in <italic class="em">de novo </italic>methylation of the previously non-methylated DNA&#160;<span class="xref">(30)</span>. Because of covalent binding of the methyl group to cytosine, the methylated state is fairy stable. S-adenosyilemthionine is a methyl group donor&#160;<span class="xref">(31)</span>. Methylated cytosines are mostly found in the cytosine-phosphate-guanosine (CpG) dinucleotide sequence. There are 28 million CpG loci in the human genome. 80 % of these being methylated. CpG dinucleotide-rich regions are referred to as CpG islets. In the human genome, mostly in promoter regions, there are approximately 45,000 CpG islets, which are usually 1000 to 2000 bases long. CpG islets generally lack DNA methylation and serve as transcription-factor binding sites. Transcription-factor binding sites are spread accross the entire genome, yet with the exception of CpG-islets they are methylated in promoter regions&#160;<span class="xref">(32)</span>.The addition of methyl groups changes the DNA biophysical properties with the resulting reduced gene expression due to inhibited binding of transcriptional activators. Another mechanism underlying reduction of gene expression is the recruitment of methyl-CpG-binding proteins, which bind to DNA and transcriptional corepressors binding to methyl-CpG-binding proteins. Demethylation may occur through active or passive mechanisms. Active demethylation is regulated by ten-eleven (TET) enzymes (named for a translocation between chromosomes 10 and 11) and thymine DNA glycosylase ( TDG), which replace methylated cytosines with non-methylated cytosines through a process of oxidation, cutting out of oxidized methylcytosine forms and through additional correction mechanisms. The passive demethylation process takes place in the absence of methylation of newly synthetised strands during DNA replication. DNA methylation is a normal cellular process, therefore changes in DNA methylation patterns may lead to DNA damage and numerous pathological conditions&#160;<span class="xref">(</span><span class="xref">33,34)</span>. In the central nervous system, DNA plays a critical role in many basic cellular processes and complex behaviours, particularly in synaptic plasticity which affects memory, learning and other cognitive processes. DNA methylation is involved in dynamic changes in gene expression, needed for long-term changes in plasticity. It is supposed to be associated with schizophrenia, addiction, depression, stress-related disorders and suicidal behaviour&#160;<span class="xref">(35)</span>. Some of the agents available for the treatment of neuropsychiatric disorders affect alterations in the DNA methylation pattern&#160;<span class="xref">(36)</span>. Studies mainly conducted on animal models and cell lines derived from humans explore the role of the following medications in the treatment of bipolar disorder, schizophrenia and depression : olanzapin ( 37, 38), lithium&#160;<span class="xref">(39)</span>, tricyclic antidepressants amytriptiline&#160;<span class="xref">(40,41)</span> and imipramine,&#160;<span class="xref">(42)</span>, escitalopram&#160;<span class="xref">(43)</span> and sodium butyrate&#160;<span class="xref">(44)</span>.</p>
			<h2><span>2.3 </span>Histones and histone modification</h2>
			<p>Histones are core building blocks of chromatin acting as spools around which DNA winds forming nucleosomes. A nucleosome is comprised of eight histone subunits (two histones H2A, H2B, H3 and H4) around which a 147 base pairs-long DNA segment wraps, and of one linker histone H1. With the exception of H1, all histones have a central globular domain and a N-terminal tail of variable lengths, on which post-translational modifications of amino acid residues are located&#160;<span class="xref">(45)</span>.The structure of nucleosomes was first proposed by Kornberg and Thomas in 1947, who defined it as a heterodimer made up of H3-H4 units with two associated dimers H2A-H2B attached to two sides&#160;<span class="xref">(46)</span>. The central globular domain consists of three alpha helices, linked by two loops&#160;<span class="xref">(47)</span>. DNA binds to histones using phosphodiester bonds. The linkeage is through three to six hydrogen bonds and is therefore independent of the DNA sequence&#160;<span class="xref">(</span><span class="xref">48)</span>. Even though histones show a high level of evolutionary conservation, there exist several histone variants, which differ in their action and stability. H3 and H4 show a high degree of conservation and have a small number of variants.H2A has the highest number of known variants, which differ from each other by the C-terminal tail and their genomic location&#160;<span class="xref">(49)</span>.</p>
			<p>Amino acid residues of the N-terminal histone tail are subject to various posttranslational modifications, involved in mitosis, meiosis, transcriptional activation and repression, DNA repair and chromatin structure modification. The most extensively studied among them are acetylation and methylation of lysines and arginines, phosphorylation of serines and threonines, ubiquitination and sumoylation of lysines and ADP-ribosylation&#160;<span class="xref">(50)</span>. Enzyme-catalysed modifications can be roughly divided into several families, including histone acetyltransferases, histone deacetylases, histone methyltransferases and histone kinases. Numerous enzymes show high specificity for amino acids situated at a well-defined position of the N-terminal tail&#160;<span class="xref">(51)</span>. Some drugs used in the treatment of mental disorders are known to inhibit histone deacytilases, e.g. valproate and valporic acid, which are used to manage bipolar disoders and, along with other medications, to treat schizophrenia. A number of pre-clinical and clinical studies are underway to determine the effects of deacetylase inhibitors on depression. In addition to the agents mentioned above, the drugs studied include sodium butyrate, vorinostat and entinostat&#160;<span class="xref">(</span><span class="xref">52-56)</span>.</p>
			<p>There are two well-known mechanisms through which histone modifications exhibit epigenetic effects on chromatin structure and function. One mechanism involves acetylation and phosphorylation of N-termina tails. The modifications neutralise the positive charge on histones, which results in changed binding affinity of the negatively charged DNA and loosening of the bond&#160;<span class="xref">(57)</span>. Another mechanism involves the activity of histone post-translational modifications resulting in the occurrence of well recognisable binding sites and protein-protein interaction&#160;<span class="xref">(51)</span>.</p>
			<p>In addition to the effects of individual histone modifications, effects exerted by histone modifications acting in combinations are important as well. Proper temporal order of specific modifications leads to changes in chromatin states. Modifications may occur within a single histone or as a combination of several histone modifications.</p>
			<p>Open and transcriptionally active chromatin markers include acetylation and trimethylation of lysine 4, lysine 36 and lysine 79 on histone 3. Decreased expression is associated with a low degree of acetylation and a high degree of methylation of lysine 9 and lysine 27 on histone 3, and of lysine on histone 4&#160;<span class="xref">(58)</span>. The effect of modification depends on the cell condition and on the combination of other epigenetic modifications, e.g. variable effects of methylation of lysines on histone 3. Lysine 9 methylation has an inhibitory effect on expression, but can also activate the expression of genes adjacent to the histone. Acetylation can either activate of inhibit gene expression, depending on other modifications present&#160;<span class="xref">(51,57)</span>. Gene transcription is therefore not dependent on a single modification, but on final conditions and a combination of modifications. Histone modifications have been implicated in the development of several disorders&#160;<span class="xref">(59)</span>, including depression, schizophrenia and bipolar disorder&#160;<span class="xref">(</span><span class="xref">60)</span>.</p>
			<h1><span>3 </span>Epigenetics of suicidal behaviour</h1>
			<h2><span>3.1 </span>Suicide and miRNA</h2>
			<p>Brain-derived neurotrophic factor (BDNF) and its receptor tropomyosin receptor kinase B) (TrkB) have been widely investigated for their role in the development of mental disorders. Research has shown that suicide victims had reduced expression of TrkB.In 2012, Maussion et al. studied miRNA expression in the prefrontal cortex (Brodmann area 10) of male controls and suicide victims with previously confirmed decrease in TrkB-T1 expression. (TrkB-T1 is a specific receptor lacking a thyrosine kinase domain 2, predominantly expressed in astrocytes, one of three NTRK2 gene (neurotrophic receptor tyrosine kinase 2) isoforms. The authors noted increased expression of hsa-miR-185* (complementary miRNA strand) and hsa-miR- 491–3p in suicide victims with low expression of TrKB-T1. Using <italic class="em">in </italic><italic class="em">silico</italic><italic class="em"> </italic>modelling, they identified five potential binding sites for miRNA hsa-miR-185* in the 3’UTR sequence of TrKB-T1, but none for hsa-miR-491–3p. Similarly, increased hsa-miR-185* expression in the prefrontal cortex was confirmed in a larger, randomly selected group of suicide completers who also showed an inverse relationship between hsa-miR-185* and TrkB-T1 expression levels. Further studies on the cerebellum showed that the results were not generalisable to other parts of the brain and thereby confirmed the specific expression of miRNA&#160;<span class="xref">(61)</span>. A global decrease in miRNA expression was found in the prefrontal cortex (Brodmann area 9) of depressive individuals who died by suicide. The expression of 21 miRNAs was significantly lowered. miRNAs, including hsa-miR-142–5p, hsa-miR-13, hsa-miR-489, hsa-miR-148b, hsa-miR-101, hsa-miR-324–5p, hsa-miR-301a, hsa-miR-146a, hsa-miR-335, hsa-miR-494, hsa-miR-20b, hsa-miR-376a*, hsa-miR-190, hsa-miR-155, hsa-miR-660, hsa-miR-130a, hsa-miR-27a, hsa-miR-497, hsa- miR-10a, hsa-miR-20a and hsa-miR-142–3p were found to be often involved in cellular differentiation and growth. Validated targets for some of these miRNAs are available in the online database <a href="http://c1.accurascience.com/miRecords">http://c1.accurascience.com/miRecords</a>. Validated targets include numerous transcriptional and growth factors, such as hsa-miR-137 E2F6 (E2 factor 6)&#160;<span class="xref">(62)</span>, proteins involved in synaptic plasticity and neurotransmitters, DNA-methyltransferase (hsa-miR-148b&#160;<span class="xref">(63)</span>, nuclear proteins and transmembrane proteins, and proteins implicated in hsa-miR-137 SDK6 (cycline-dependent kinase 6) signalling&#160;<span class="xref">(62)</span>. A group of 29 miRNAs showed a high level of co-regulation of expression in suicide victims, but not in controls, which suggests additional reorganisation of miRNA pathway&#160;<span class="xref">(64,65)</span>.</p>
			<p>In their preliminary studies conducted in 2014, Lopez et al. confirmed the role of polyamine system in suicidal behaviour. Polyamine system is involved in many cellular processes, particularly in the modulation of cell cycle and ion channels, as well as in the stress response. Polyamines, including putrescine, spermine, spermidine and agmatine, interact with a negatively charged DNA and thereby affect gene expression&#160;<span class="xref">(66)</span>. Suicide victims showed reduced expression of SAT1 gene (spermidine/spermine Ni-acetyltransferase) and spermine oxidase (SMOX). Further studies were undertaken to determine whether decreased expression of both genes in the prefrontal cortex (Brodmann area 44) of depressed suicide victims is involved in miRNA regulation. Expression levels of ten miRNAs were studied, i.e. of miR-124a, miR-139–5p, miR-195, miR-198, miR-320c, miR-33b, miR-34a, miR-34c-5p, miR-497 and miR-873, showing appropriate sequence for binding to 3’UTR of mRNA of both genes, identified by computer-based analysis. Suicide completers had higher expression levels of the following four miRNAs: miR-34c-5p, miR-139–5p, miR-195 and,miR-320c. These miRNAs have hithero been investigated in cancer patients and in individuals with neurodegenerative disorders. A negative correlation was noted between two miRNAs (SAT1-miR-34c-5c, and miR-320c) and gene expression levels. Both genes were seen to be negatively correlated with miR-320c, which is, according to the data from the miRBase, the only poorly conserved miRNA during evolution, and is expressed only in higher primates&#160;<span class="xref">(67)</span>.</p>
			<p>SAT1 expression in depressed victims of suicide was explored by Pantazatos et al. (2017). No differences in miRNA expression levels in the prefrontal cortex (Bormann area 9) were found between controls, depressed suicide completers and individuals with depresssion who did not die by suicide. Both groups of depressed individuals had low levels of SAT1 expression yet they were not associated with miRNA&#160;<span class="xref">(68)</span>. Further studies were carried out in the same brain region of the same groups of individuals. Whole transcriptome analysis with total RNA sequencing detected changes in the level of expression of 35 genes. In one group of participants, RNA sequencing was used to assess the miRNA expression profile, yet it detected no changes in miRNA expression levels&#160;<span class="xref">(69)</span>.</p>
			<p>A recent study demonstrated changes in miRNA expression in the <italic class="em">locus coeruleus,</italic> a nucleus in the brainstem and the principal site for brain synthesis of noradrenaline. A comparison of controls and depressed suicide victims revealed 13 miRNAs with changed expression. In the group of depressed suicide victims there were ten miRNAs with increased levels of expression: hsa-miR-17–5p, hsa-miR-20b-5p, hsa-miR-106a-5p, hsa-miR-330–3p, hsa- miR-541–3p, hsa-miR-582–5p, hsa-miR-890, hsa-miR-99b-3p, hsa-miR-550–5p and&#160;&#160;hsa-miR-1179. Expression levels were reduced in three miRNAs : hsa-miR-409–5p, hsa-1et-7g-3p and hsa-miR-1197. Using the <italic class="em">in silico</italic><italic class="em"> </italic>approach, the investigators defined potential miRNA target sites and the corresponding miRNAs, forming a complex regulatory network. The following pathways that are often implicated in depression and suicidal behaviour were identified: signalling of protein kinase A, phospholipase C, glucocorticoids, signal-regulated (ERK) mitogen- activated protein kinase (ERK/MAPK) signalling pathway, neutrophil-related factors, GABAergic signalling and glutamatergic signalling pathway. The obtained regulation network data were tested on samples in order to determine the expression of some predicted protein-coding target genes. In suicide victims, expression levels were reduced in the following genes associated with depression and suicidal behaviour: relin (RELN), glycogen synthase kinase 3β (GSK-3β), monoamine oxidase A (MAOA), muscarinic acetylcholine receptor (CHMR1) and 1-phosphatidylinositol-4.5-biphosphate beta-1 (PLCB1)&#160;<span class="xref">(70)</span>.</p>
			<h2><span>3.2 </span>Suicide and DNA methylation</h2>
			<p>Some post-mortem studies of brain tissue from suicide completers showed a relationship between epigenetic factors and suicidal behaviour.The advent of next-generation (massive) sequencing technology has accelerated sequencing of DNA sequences, and rendered it more accessible&#160;<span class="xref">(71)</span>. This new method allowed for the study of global DNA methylation patterns. Labonte et al. (2013), who investigated DNA methylation patterns in the hypocampus, identified 366 promoters that were differentially methylated in suicide victims relative to control subjects (both hypermethylated and hypomethylated). Numerous promoters belonged to genomes involved in cognitive processes&#160;<span class="xref">(72)</span>. The analysis of global DNA methylation patterns of the prefrontal cortex revealed increased methylation in depressed suicide completers relative to controls&#160;<span class="xref">(73)</span>. Schneider et al. (2015) investigated DNA methylation patterns in the prefrontal cortex of male suicide completers. Individuals who died by suicide showed lower global methylation accross the entire genome.A comparison of different methylated regions revealed higher DNA methylation in suicide completers relative to controls&#160;<span class="xref">(74)</span>. In another study, astrocyte markers, i.e. genes with increased expression in the astrocytes, the star-shaped glial cells, were studied in depressed suicide victims.In the prefrontal cortex of these individuals whole-genome analysis showed different patterns of reduced methylation in differently methylated DNA regions&#160;<span class="xref">(7)</span>.</p>
			<p>Some other investigations have focused on methylation patterns of selected candidate genes. One of the first studies was carried out on Slovene participants. Investigation of BDNF promoter methylation levels in the Wernicke area showed increased methylation levels in some CpG-dinucleotides. Increased DNA methylation was reflected in decreased levels of mRNA BDNF expression&#160;<span class="xref">(75)</span>.</p>
			<p>In some other studies, the investigators found elevated methylation levels of rRNA (ribosomal RNA) promoter in the hippocampus&#160;<span class="xref">(76)</span>, of HTR2A (serotonin type 2A receptor) in the frontal lobe&#160;<span class="xref">(77)</span>, and of TrkB gene promoter in the frontal cortex&#160;<span class="xref">(78)</span> of individuals who died by suicide.</p>
			<p>Since DNA methylation is tissue-specific, methylation patterns are investigated in disease-relevant tissues. One of the major drawbacks of epigenetic studies is that samples from different brain areas can be obtained and investigated only after death. Therefore, more accessible peripheral tissues, e.g.blood, have been used in epigenetic studies to determine how changes in blood mirror CNS processes. Global methylation analysis of several areas of the brain of healthy subjects showed several correlations of DNA methylation levels between blood and brain areas&#160;<span class="xref">(79)</span>.</p>
			<p>Similarly, in animal models a correlation was found between cytosine methylation patterns in the brain and peripheral blood. The methylation status of specific sites in the blood could therefore be used as a biomarker with potential clinical value&#160;<span class="xref">(80)</span>.</p>
			<h2><span>3.3 </span>Suicide and histone modification</h2>
			<p>Only a few studies so far have addressed the relationship between suicidal behaviour and histone modifications. Ernst et al.(2009) used animal models of depression to explore histone 3 lysine 27 ( H3K27) methylation in the orbital frontal cortex (Brodmann area 10 – BA 10) and cerebellum. Suicide victims showed increased levels of histone 3 lysine 27 methylation in BA 10. Lysine methylation was negatively correlated with TrkB-T1 expression&#160;<span class="xref">(81)</span>.</p>
			<p>In 2010 Fiori et al. studied histone modifications related to polyamine system. In addition to identifying polymorphisms in the promoter area and DNA methylation pattern, they determined the level of histone 3 lysine 27 methylation in the prefrontal cortex (Brodmann areas 8 and 9, BA8 and BA 9) for two genes, SMS and SMOX. No differences in histone 3 lysine 27 methylation pattern in the promoter area of the two genes were observed between suicide completers and control subjects. Nor was there any difference between lysine methylation and gene expression or DNA methylation&#160;<span class="xref">(82)</span>.</p>
			<p>Further studies were undertaken to explore decreased gene expression of SAT1 mRNA, established in previous investigations. No differences in the level of histone 3 lysine 27 trimethylation in the prefrontal cortex (BA9 and BA10) were seen between suicide victims and controls, nor was there any relationship with decreased SAT1 expression&#160;<span class="xref">(83)</span>.</p>
			<p>Based on the previously found increase in the expression of four polyamine genes, histone 3 lysine 27, trimethylation was investigated for the following genes: ARG2 (arginase II), AMDI (S-adenosylmethionine dexarboxylase), OAZ1 and OAT2 (ornithine dexarboxylase antimycin 1 and 2). Increased trimethylation of lysine 4 on histone 3 in the promoter region of OAZ1 was confirmed in suicide victims.</p>
			<h1><span>4 </span>Conclusion</h1>
			<p>Suicidal behaviour is a complex phenomenon and is not yet fully understood. The advent of new technologies made it possible to study epigenetic changes, which together with the more thoroughly investigated genetic background, help us to better understand hereditary factors involved in suicidal behaviour. Research has shown a strong interconnection of spatial and temporal regulation of miRNA expression, DNA methylation and histone post-translational modifications. These form extensive regulatory networks, which can modify cell signalling and thereby contribute to the development of mental disorders. Thanks to its dynamic properties, the epigenome could, in theory, be modified, and is therefore a potential pharmacological target. Epigenetic studies can explain only some of the changes in gene expression. Though these represent only small pieces in the mosaic of biological factors, they play a critical role in the development of mental disorders.</p>
			<h1 class="ack"><span>5 </span>Acknowled<a id="_idTextAnchor000"></a>gement</h1>
			<p>The research for this paper, i.e. programme PI-0390, project 13–7132 and junior investigator grant, was financially supported by the Agency of the Republic of Slovenia for Research.</p>
		</article>
	</body>
</html>
